public/app/features/alerting/unified/components/silences/SilencePeriod.tsx

```diff
@@ -0,0 +1,78 @@
+import { css } from '@emotion/css';
+import { dateTime, GrafanaTheme } from '@grafana/data';
+import { Field, TimeRangeInput, useStyles } from '@grafana/ui';
+import React from 'react';
+import { useController, useFormContext } from 'react-hook-form';
+import { SilenceFormFields } from '../../types/silence-form';
+
+export const SilencePeriod = () => {
+  const { control, getValues } = useFormContext<Pick<SilenceFormFields, 'startsAt' | 'endsAt' | 'timeZone'>>();
```

Comment:
is there a reason to `Pick` those few fields?

Reply:
Nope, I thought about creating a separate type for the silence period fields but there's really no point.